linux/tests/CMakeLists.txt

```diff
@@ -2,21 +2,19 @@
 
 # unit tests
 if(${run_unittests})
-    build_test_folder(async_socket_linux_ut)
-    build_test_folder(completion_port_linux_ut)
-    build_test_folder(gballoc_ll_passthrough_ut)
-    build_test_folder(gballoc_hl_passthrough_ut)

```

Comment:
These should not be removed

Reply:
You might need to update your branch from master

Reply:
done